### PR TITLE
Create Help Page Structure

### LINKED ISSUE #73: 🙋‍♀️ Help Page
**Type**: client
**FOF**:  2

## Description
- Build out the mobile and desktop views for the help page
- Utilize various base UI components from Amit's PR (#65)

## Screenshot
image
image

## Todo

- [ ] Build out a component that display the content for each resource
- [ ] Iterate through mock data to display list of resources under multiple sections
- [ ] Make the help page both desktop and mobile compatible

## Links
- [Figma](https://www.figma.com/file/6aJ3cfu2Yq5dxUQcNDkm6Y/Brink-Designs?node-id=34%3A2)

### PR DESCRIPTION
**Pull request by suewee**
## Status:

:rocket: Ready

## Summary
Create the Help page.

Closes #73 

## Changes

- Change Navbar height, add black bar outside of landing & home page
- Create the Help Page

### Desktop
image

### Mobile
image

### CODE REVIEW

File: client/src/components/help/ResourceItem.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export const ResourceItem: FC<ResourceItemProps> = ({
+  title = '',
+  description = '',
+  linkText = '',
+  linkHref = '',
+  isExternalLink = false,
+  ...rest
+}): ReactElement => (
+  <Alert
```
Comment (amit-sawhney) [inline client/src/components/help/ResourceItem.tsx:24]: This component is a more of an Alert component. Yes it pertains to the help page, but I think a better folder name for this component to live in is called Alert

File: client/src/pages/Help.tsx [hunk 2]
```diff
@@ -1,23 +1,71 @@
...
+          <Heading as="h3" mt={3} fontSize="22px">
+            Section Name
+          </Heading>
+          <List spacing="10px" mb={6}>
+            <ResourceItem
+              title="How to handle polling place issues"
+              description="Below are some of the typical problems voters face with information on what you should say to advocate for yourself."
+              linkText="Check potential polling place issues"
+              linkHref="/about"
+              isExternalLink
```
Comment (amit-sawhney) [inline client/src/pages/Help.tsx:37]: This is really good design -- I like

File: client/src/pages/Help.tsx [hunk 3]
```diff
@@ -1,23 +1,71 @@
...
+            <ResourceItem
+              title="File a voting or polling complaint"
+              description="Below is a link to file the voting or polling complaint."
+              linkText="File a complaint"
+              linkHref="/about"
+            />
+          </List>
+        </section>
+
+        <section>
```
Comment (amit-sawhney) [inline client/src/pages/Help.tsx:48]: Good use of <section> tags for user accessibility!

**Review by amit-sawhney — APPROVED** [review]
Looks smexy -- This will be nice to start rounding off stuff with and getting contentful involved :)

File: client/src/layout/Navbar.tsx [hunk 4]
```diff
@@ -26,6 +26,8 @@ export const Navbar = (): ReactElement => {
   const shouldShowAddress = () => address && location.pathname !== '/landing';
   const shouldShowFullLogo = () =>
     isDesktop || location.pathname === '/landing';
+  const shouldShowBottomBar = () =>
```
Comment (ayanmallik) [inline client/src/layout/Navbar.tsx:29]: Thanks for handling this styling in your PR!

File: client/src/layout/Navbar.tsx [hunk 5]
```diff
@@ -70,8 +72,10 @@ export const Navbar = (): ReactElement => {
         listStyleType="none"
         backgroundColor="baseLight"
         alignItems="center"
-        height="100px"
+        height="80px"
```
Comment (ayanmallik) [inline client/src/layout/Navbar.tsx:75]: After Bryan/Jocelyn's feedback presentation on Monday, we might want to look into decreasing the height of the navbar even more.

File: client/src/pages/Help.tsx [hunk 6]
```diff
@@ -1,23 +1,71 @@
...
+          </Heading>
+          <List spacing="10px" mb={6}>
+            <ResourceItem
+              title="How to handle polling place issues"
+              description="Below are some of the typical problems voters face with information on what you should say to advocate for yourself."
+              linkText="Check potential polling place issues"
+              linkHref="/about"
+              isExternalLink
+            />
+            <ResourceItem
```
Comment (ayanmallik) [inline client/src/pages/Help.tsx:39]: Totally agree with Amit, the modularity with your ResourceItem is wonderful - chef's kiss!

**Review by ayanmallik — APPROVED** [review]
Great work Sue, the Help Page looks identical to the designs on Figma and I love the way you modularized the resources on the page! Let's look into merging this PR during tonight's dev night

**Pull request merged**